### ISSUE TITLE
.coala taking priority over command-line arguments

### ISSUE THREAD

**Issue by Seanny123** [coala/coala#6086]
I have created a repo to demonstrate this error:
https://github.com/Seanny123/coala-fail

Expected Behaviour
----------------------------

`coala -f=cfail/check.py` would only run bears on the file `check.py`.

Actual Behaviour
------------------------

In `.coala`, the rule `files=**.py` takes priority over the command line argument `-f=cfail/check.py`. Thus, both `check.py` and `ignore.py` are checked.

Work-Around
------------------

Anytime I want to verify an individual file, I have to delete the `files=` line from `.coala`. This is a bit inconvenient and goes against my expectations.

In this intended behaviour or a bug?

**Comment by codehobbyist06** [comment]
Yes, I have reproduced this issue and it persists. What actually happens is coala first runs the settings given in the coafile and then the settings given by cli(I have checked this by running coala in debug mode). According to the coala documentation this is a bug because it says that, 

> Settings given in the project-wide coafile override all settings given by other files and can only be overridden by settings given via the command line interface

Can someone from coala maintainers pls tell how this issue can be resolved?
According to me, I first need to check the flow of programs to find the file that is responsible for this behaviour. Can someone help with this?
